waggle-dance-api/src/test/java/com/hotels/bdp/waggledance/api/model/PrimaryMetaStoreTest.java

```diff
@@ -30,6 +32,11 @@
 
 public class PrimaryMetaStoreTest extends AbstractMetaStoreTest<PrimaryMetaStore> {
 
+  String name = "name";
```

Comment:
Make all of these private. We generally aim to keep the access level of everything as closed as possible.